README.md

```diff
@@ -55,7 +55,7 @@ Then, in your `.eslintrc.json`:
 
 * The first option:
 
-  - Objects are passed directly to Prettier as [options](https://github.com/prettier/prettier#api). Example:
+  - Objects are passed directly to Prettier as [options](https://github.com/prettier/prettier#api). This plugin will merge and override any config set with a `.prettierrc` file (only for prettier >= 0.17). Example:
```

Comment:
- Here's a better link for options: https://github.com/prettier/prettier/#options
- You could change "prettier" to "Prettier" for consistency :)
- Typo: `0.17` → `1.7.0`
- I think it is clearer to say `[Prettier configuration file](https://github.com/prettier/prettier/#configuration-file)` instead of `` `.prettierrc` file ``. Not sure if we need to mention the version at all.
- The sentence about config file support applies both to the object option and the `"fb"` option. I suggest moving it outside of all the bullet lists to the very start of the "Options" section.

Reply:
I fixed the typos, changed the link and put the sentence about config files outside the sublist, but at the end. WDYT?